official/mnist/mnist.py

```diff
@@ -233,10 +225,11 @@ def eval_input_fn():
 
   # Export the model
   if FLAGS.export_dir is not None:
-    image = tf.placeholder(tf.float32,[None, 28, 28])
-    serving_input_fn = tf.estimator.export.build_raw_serving_input_receiver_fn(
-        {"image_raw":image})
-    mnist_classifier.export_savedmodel(FLAGS.export_dir, serving_input_fn)
+    image = tf.placeholder(tf.float32, [None, 28, 28])
```

Comment:
Why is image here and another placeholder is created in the dict? I think this one is left over from an earlier life.

Reply:
Done